R/sortable_options.R

```diff
@@ -158,6 +164,7 @@ sortable_options <- function(
   handle = NULL,
   filter = NULL,
   draggable = NULL,
+  swap = NULL,
```

Comment:
Let's move this to the top. Let's add `multiDrag` as well.

Let's also move `...` to the beginning?

Reply:
Agreed.